docs/src/getting-started/installation.md

```diff
@@ -329,6 +330,76 @@ Make sure to include this section in `Scarb.toml`
 test = "snforge test"
 ```
 
+## Set up shell completions (optional)
+
```

Comment:
nit: Maybe I'm a bit too sensitive, but could we add just a sentence about what completions are for 😄 ? Wdyt?